src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,401 @@
+---
+title: Easy OTP on the web with the SMS Receiver API
+subhead: Help users type OTP received through SMS
+authors:
+  - agektmr
+
+date: 2019-09-19
+# updated: 2019-06-27
+
+hero: hero.png
+# You can adjust the fit of your hero image with this property.
+# Values: contain | cover (default)
+# hero_fit: contain
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+# You can provide an optional cropping of your hero image to be used as a
+# thumbnail. Note the alt text will be the same for both the thumbnail and
+# the hero.
+# thumbnail: thumbnail.jpg
+alt: A description of the hero image for screen reader users.
```

Comment:
```suggestion
alt: A cartoon of a man receiving a one-time password on his phone and using it to log in on his computer.
```

Reply:
The API is not available for the desktop at this point, so I will probably replace the image with something else.